package.json

```diff
@@ -14,7 +14,8 @@
   "main": "index.js",
   "scripts": {
     "lint": "xo",
-    "test": "xo && nyc mocha test/**/*.js"
+    "test": "xo && nyc mocha test/**/*.js",
+    "semantic-release": "npx semantic-release -e ./config/release.config.js"
```

Comment:
So should our CircleCI script be calling `npm run semantic-release` instead of `npx semantic-release`...?

Reply:
I don't entirely understand what this is supposed to be for. It's included because the [README](https://github.com/JPeer264/node-semantic-git-commit-cli#usage) for semantic-git-commit-cli said to include it. I added the `npx` because I deviated from the README so we could install it and run it from the local node_modules via `npx`. I just tested without it and it works so I'll just remove it.

Reply:
If you have something in the config file you added that we need for semantic-release, then you need to change `.circleci/config.yml` to call this npm script instead of just the `npx` version or it won't see that config. But why do we need that config, anyway? What do we get out of this `config/release.config.js`?